draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -938,6 +1022,14 @@
       receiving a packet destined for the Service ID, the ingress router must
       refresh the Timer. The ingress router must send a BGP Withdraw UPDATE
       for the Service ID upon expiration of the Timer.</t>
+      <!-- XXX JMH - this text regarding RT-Constrain is very confusing.  RTC
+      operates on BGP routes having route target extended communities.  This
+      document isn't suggesting to use the existing route targets defined in
+      RFC 4360 and using that type/subtype is probably inappropriate.  It'd be
+      better to define a new extended community for this feature. 
```

Comment:
Is it better to apply a new Route Target Code Point  for the purpose of constraining Metadata Path Attribute distribution? like getting a new SAFI?

Currently the SAFI =132 is for Route Target Constrains.
Section 4 of the RFC4684 states that “Route targets can then be expressed as prefixes”. Therefore, it should be okay to use the same SAFI =132 for Ingress routers to send out the Route Target Membership NLRI Advertisements for the interested service IP prefix. 

Is there any problem?